lesson_9/ilas_lorena/classes homework.py

```diff
@@ -0,0 +1,81 @@
+# Build two classes of your choice that can model a real-life example. The class needs to meet the following
+# requirements:
+# at least 5 attributes each at least 2 methods each one class to inherit from another As a demonstration create at
+# least 5 instances of one class (preferably the child class) and call all the methods it holds
+
+
+class Animal(object):
+
+    def __init__(self, animal, name, n_legs, gender, speech_type):
+        self.animal = animal
+        self.name = name
+        self.n_legs = n_legs
+        self.gender = gender
+        self.speech_type = speech_type
+
+    def check_speech_type(self):
+        if self.speech_type == 'meow':
+            return 'It is a cat!'
+        else:
+            return f'{self.name} is a {self.animal} so it does not speak meow'
+
+    def check_motility(self):
+        if self.n_legs == 4:
+            return f'{self.name} the {self.animal} has 4 legs so he can walk'
+        elif self.n_legs == 0:
+            return f'{self.name} the {self.animal} has no legs so it slides'
+        else:
+            return f'{self.name} has {self.n_legs} legs so maybe it jumps maybe it flies who knows'
+
+
+class Dog(Animal):
+
+    def __init__(self, animal, name, n_legs, gender, speech_type, breed, age):
+        self.breed = breed
+        self.age = age
+        super().__init__(animal, name, n_legs, gender, speech_type)
+
+    def check_breed(self):
+        if self.breed[0:1] in ('B', 'G', 'L', 'C'):
```

Comment:
Since you need just one letter you can use index instead of slice. `self.breed[0]` will work just the same

Reply:
Thank you, I changed the index as you suggested :)